encrypted-media-respec.html

```diff
@@ -441,6 +445,10 @@
               (The <a def-id="distinctive-permanent-identifier-maybe-plural"></a> MUST NOT ever be provided to the application, even in opaque or encrypted form.) 
             </p>
           </div>
+          <p class="note">
+            <a def-id="option-distinctiveIdentifier"></a> controls whether Distinctive Identifiers may be used.
+            Specifically, Distinctive Identifiers may only be used when the value of the <a def-id="option-distinctiveIdentifier"></a> member of the <a>MediaKeySystemAccess</a> used to create the <a>MediaKeys</a> object is <a def-id="requirement-required"></a>.
+          </p>
```

Comment:
My first instinct was to agree with this comment, but the notes explicitly apply the requirement for Distinctive Identifiers, Distinctive Permanent Identifiers, and user of identifiers.  To me, the current approach seems preferred.